### PR TITLE
Added access rights for streams, stream views, types, quantities, and…

### PR DESCRIPTION
**Pull request by Annamn61**

### CODE REVIEW

File: Documentation/SequentialDataStore/SDS_Streams.md [hunk 1]
```diff
@@ -613,3 +613,37 @@ The response includes a status code.
 ```csharp
    Task UpdateStreamOwnerAsync(string streamId, Trustee streamOwner);
 ```
+*** 
+
+## `Get Stream Access Rights`
+
+Gets the AccessRights associated with the specified stream for the requesting identity.
```
Comment (mckenzielee) [inline Documentation/SequentialDataStore/SDS_Streams.md]: Do we have documentation on the access rights and what they mean that we can link? Should we include this information?

File: Documentation/SequentialDataStore/SDS_Streams.md [hunk 2]
```diff
@@ -613,3 +613,37 @@ The response includes a status code.
...
+The namespace identifier  
+  
+`string streamId`  
+The stream identifier  
+
+**Response**  
+The response includes a status code and a response body.
+
+**Response Body**  
+The access rights associated with specified stream for the requesting identity.
```
Comment (mckenzielee) [inline Documentation/SequentialDataStore/SDS_Streams.md]: Do you think an example response makes sense?

File: Documentation/SequentialDataStore/SDS_Streams.md [hunk 3]
```diff
@@ -613,3 +613,46 @@ The response includes a status code.
 ```csharp
    Task UpdateStreamOwnerAsync(string streamId, Trustee streamOwner);
 ```
+*** 
+
+## `Get Stream Access Rights`
+
+Gets the AccessRights associated with the specified stream for the requesting identity. For
```
Comment (mckenzielee) [inline Documentation/SequentialDataStore/SDS_Streams.md]: Any specific reason that AccessRights is one word here and two words in the next sentence?

Comment (Annamn61) [reply Documentation/SequentialDataStore/SDS_Streams.md]: Just getting a little too used to coding "AccessRights" as one word, there definitely should be a space here

**Review by mckenzielee — APPROVED** [review]

**Pull request merged**